packages/api/src/controllers/stripe.ts

```diff
@@ -699,4 +700,122 @@ app.post("/migrate-pro-user", async (req, res) => {
   });
 });
 
+app.post(
+  "/subscribe-to-enterprise",
+  authorizer({ anyAdmin: true }),
```

Comment:
Should this be here or under the `/user` APIs? Where is the API to change plans today?